src/CommunityToolkit.Maui/Views/RatingView/RatingView.Shared.cs

```diff
@@ -0,0 +1,361 @@
+﻿using System.Windows.Input;
+using CommunityToolkit.Maui.Core.Primitives;
+using CommunityToolkit.Maui.Core.Primitives.Defaults;
+using Microsoft.Maui.Controls.Shapes;
+
+namespace CommunityToolkit.Maui.Views.RatingView;
+
+/// <summary>RatingView control.</summary>
+public class RatingView : Border
+{
+
+	Microsoft.Maui.Controls.Shapes.Path[] shapes;
+
+	string shape = string.Empty;
+
+	Grid innerContent = new();
+
+
+	/// <summary> Rating value bindable property <see cref="CurrentRating"/> </summary>
+	public static readonly BindableProperty CurrentRatingProperty = BindableProperty.Create(nameof(CurrentRating), typeof(double), typeof(RatingView), defaultValue: 0.0, propertyChanged: OnBindablePropertyChanged);
+
+
+	/// <summary>MaximumRating rating value Bindable property </summary>
+	public static readonly BindableProperty MaximumRatingProperty = BindableProperty.Create(nameof(MaximumRating), typeof(int), typeof(RatingView), defaultValue: 5, propertyChanged: OnBindablePropertyChanged);
+
+
+	/// <summary>Star size Bindable property</summary>
+	public static readonly BindableProperty SizeProperty = BindableProperty.Create(nameof(Size), typeof(double), typeof(RatingView), defaultValue: 20.0, propertyChanged: OnBindablePropertyChanged);
+
+
+	/// <summary>Star Color Bindable property.</summary>
+	public static readonly BindableProperty FilledBackgroundColorProperty = BindableProperty.Create(nameof(FilledBackgroundColor), typeof(Color), typeof(RatingView), defaultValue: Colors.Yellow, propertyChanged: OnBindablePropertyChanged);
```

Comment:
What is preferable Backgroud as Brush or BackgroundColor as Color?
Background allows more customization like gradient background